src/invariant/EventsAreConsistentWithEntryDiffs.cpp

```diff
@@ -431,6 +483,23 @@ aggregateEventDiffs(Hash const& networkID,
                 return std::nullopt;
             }
         }
+        else if (eventNameVal.sym() == "set_authorized")
+        {
+            if (topics.size() != 3)
+            {
+                return std::nullopt;
+            }
+
+            auto idVal = topics.at(1);
+
+            if (event.body.v0().data.type() != SCV_BOOL)
+            {
+                return std::nullopt;
+            }
+
+            res.mIsAuthorized[idVal.address()][asset] =
```

Comment:
I think it's worth adding a comment stating that the 'aggregation' logic for authorization is just 'use the latest value', as this assignment is not immediately obvious.

Reply:
Good point. Done